CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ without native/platform specific bindings and SDKs. See [this ticket for more de
 
 - Drop .NET 6 Mobile in favor of .NET 7 ([#2624](https://github.com/getsentry/sentry-dotnet/pull/2604))
 
+API Changes:
+
+- Removed IHasBreadcrumbs ([#2670](https://github.com/getsentry/sentry-dotnet/pull/2670))
```

Comment:
I guess here it's safe to say: 

`IHasBreadcrumbs` was removed. Use `IEventLike` instead. ?